core/src/sql/statements/info.rs

```diff
@@ -11,58 +11,31 @@ use serde::{Deserialize, Serialize};
 use std::fmt;
 use std::sync::Arc;
 
-#[revisioned(revision = 3)]
+#[revisioned(revision = 4)]
```

Comment:
Made this revision 4 so that the descriminants or now 0 to 5 instead of 1,3,5,7,10